bin/install-cassandra.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# This script will install Cassandra at $CASSANDRA_INSTALLATION_DIR/cassandra/ and start it. The process is as follows:
+# 1. Download apache-cassandra-$CASSANDRA_VERSION-bin.tar.gz (if not exist) and then extract it
+# 2. Start Cassandra instance locally
+# 3. Waiting the initialization to complete for 15 seconds
+
+SAVEDIR=$PWD
+# Change this to your preferred installation directory, and make sure to update cassandraRootDir at SSTableTest.java
+CASSANDRA_INSTALLATION_DIR="/tmp"
```

Comment:
Let's move these two variables in bin/lib.sh and make sure you are actually loading lib.sh; look at the other scripts to see how this is done. In this way it's only defined in one place.